api/graphql/schema.graphql

```diff
@@ -1470,7 +1475,9 @@ type Query {
     @all
     @canModel(ability: "viewAny")
   department(id: UUID! @eq): Department @find @canModel(ability: "view")
-  departments: [Department]! @all @canModel(ability: "viewAny")
+  departments(where: DepartmentFilterInput!): [Department]!
```

Comment:
Made the filter input non-nullable otherwise the scopes don't run at all

Reply:
I don't follow, searching the schema and I don't see `@spread` on filter `where`'s

Reply:
I don't love this either.  Having to add an empty query parameter everywhere is pretty ugly.  We want this change to be as transparent as possible.  Let me know if you want me to tinker with it.